templates/slurm.txt

```diff
@@ -195,11 +195,28 @@ Autoscale = $Autoscale
         [[[network-interface eth0]]]
         AssociatePublicIpAddress = $ExecuteNodesPublic
 
+    [[nodearray htc]]
```

Comment:
Can you re-order this so that git can understand the diffs rather than showing an entirely new nodearray?